plottr/plot/mpl.py

```diff
@@ -254,14 +258,22 @@ def colorplot2d(ax: Axes, x: np.ndarray, y: np.ndarray, z: np.ndarray,
         im = ppcolormesh_from_meshgrid(ax, x, y, z, cmap=cmap, **kw)
     elif style is PlotType.scatter2d:
         im = ax.scatter(x, y, c=z, cmap=cmap, **kw)
+    elif style is PlotType.linecuts2d:
+        im, sm = plot_as_linecuts(ax,x,y,z,cmap=cmap, **kw)
 
     if im is None:
         return
 
-    cax = attachColorBar(ax, im)
-    ax.set_xlabel(axLabels[0])
-    ax.set_ylabel(axLabels[1])
-    cax.set_ylabel(axLabels[2])
+    if type(im) == Line2D:
```

Comment:
the more robust/correct way of checking the type in python ;)
```suggestion
    if isinstance(im, Line2D):
```